src/Shell/Task/ModelTask.php

```diff
@@ -337,7 +340,7 @@ public function findHasMany($model, array $associations)
                         'alias' => $otherModel->alias(),
                         'foreignKey' => $fieldName
                     ];
-                } elseif ($otherTable === $tableName && $fieldName === 'parent_id') {
+                } elseif ($otherTableName === $tableName && $fieldName === 'parent_id') {
```

Comment:
I feel more confident in doing the other tests then. But the mess is due to my lowish *git-foo* working on local master branch instead of feature branches. As also a lot of time passed I will prolly just copy paste the snippets I changed *fun* (if you have a better idea, let me know).